src/main/java/com/iota/iri/BundleValidator.java

```diff
@@ -8,8 +8,40 @@
 
 import java.util.*;
 
+/**
+ * Validates bundles.
+ * <p>
+ * Bundles are a list of transactions that represent an atomic transfer, meaning that either all
+ * transactions inside the bundle will be accepted by the network, or none.
+ *</p>
```

Comment:
would be good to add a line about how bundle structure is defined. (same bh and connected by trunk)

Reply:
:+1: